### ISSUE TITLE
Give `http://melta.no` troubleshooting help link when plugin fails, with CTA to Slack in web copy

### ISSUE THREAD

**Issue by DouweM** [meltano/meltano#6645]
Spec (edited by AJ):

When Meltano fails, we'll print a standard error message:

> [Error description text.]
>
> To contact the Meltano team and access troubleshooting help and resources, visit http://melta.no

<details>
  <summary>Orig. issue description (Douwe)</summary>
  
When `meltano invoke` or `meltano run` fails, a user is more likely to manage to solve the issue and stick with Meltano if they join Slack than if they don't, as evidenced by the number of people who responded to the survey saying the docs weren't always helpful, but the Slack community was. I imagine a good amount of people that try us out and churn would not have if they had known about Slack. That also lets us reach out to them to learn more about their use case and experience, even if they do end up churning.

So I think we should recommend joining Slack in the product wherever that makes sense.

@tayloramurphy Wdyt? Seems like an easy change with potentially large impact.

</details>

**Comment by tayloramurphy** [comment]
@DouweM I'm in favor of adding it. 

@aaronsteers what do you think about this:

* Add text like `Need help fixing this problem? Join our Slack at meltano.com/slack to talk to the community and team.` that appears at the end of an error message. 
* Make this opt-out functionality b/c once I've joined Slack I'd be terribly annoyed at having it constantly ask me
  * Perhaps we can have a feature flag for new user helpers where message like this and possibly others in the future can be bundled and once a user is more onboarded they can turn them off? `ff.new_user_helpers`?
* Highlight the text in a different color than red (blue? yellow?)

**Comment by pandemicsyn** [comment]
@DouweM @tayloramurphy I think it would be nice to make this useful for folks that are a bit more savvy and just need a little guidance as well. There's also the whole, teaching a person how to fish aspect and all that jazz. 

What about doing something like:

```
Need help fixing this problem? Visit https://melta.no/ for troubleshooting steps or to join our slack community to get help fast.
```

And then that target URL can be a dedicated landing page for helping folks. In the 1st version maybe just a basic page with two CTA's. One that leads you to a basic "here's the common things to check" and one CTA for joining slack.

Over time you can develop that troubleshooting path more and more.  (i.e. doing that log analyzer we've chatted about in the path, having a common issues/resolutions collection, info on office hours/twitch streams/pairing jam sessions, etc).

I feel like this would send a much stronger "Wow these folks really care about the DevX" vibe, than just dropping a link to slack.

>as evidenced by the number of people who responded to the survey saying the docs weren't always helpful, but the Slack community was.

My take away from that isn't "we should funnel more users to slack".  My take away is that we should fix our docs and troubleshooting story.  The lack of a troubleshooting , faq, common issues, etc has been a big hole we've all chatted about before.  

If you visit docs.meltano.com after an error. You'll probably scan for a help/troubleshooting section in the side bar. Which we don't have. If you search for "troubleshoot" the first hit is:

https://docs.meltano.com/guide/installation#troubleshooting-installation , which has a giant CTA that just leads back to https://docs.meltano.com/ whos two CTA for getting help are browsing issue and joining slack. 😬

**Comment by tayloramurphy** [comment]
@pandemicsyn yeah, that's a much better idea 😄 

I'll spin out a new issue for creating the troubleshooting page. This issue can be about adding the text in the help output still. I'm in favor of using `https://melta.no/` too.

**Comment by aaronsteers** [comment]
I agree also with the proposals from @pandemicsyn. 👍

**Comment by DouweM** [comment]
@tayloramurphy @aaronsteers That troubleshooting page idea is great, but as that will take more time to realize we're still doing the Slack link in the mean time, right? That could land this week and start helping people ASAP, so I don't want to wait.

I really like having melta.no go to the troubleshooting page, but in the mean time could I suggest having it redirect to meltano.com/slack.

**Comment by tayloramurphy** [comment]
@DouweM yes, that's the plan. It's assigned to the iteration and ready to be picked up by an engineer or assigned out to one via @aaronsteers

**Comment by aaronsteers** [comment]
@tayloramurphy - I'm tried drafting this several times and I'm not able to come up with a concise-enough phrasing that (1) prints the slack info and (2) tells the user how to turn it off. If we leave off the second part, this feels super spammy, as you, I, and others noted above. Adding both, though, makes it overly verbose. Putting a troubleshooting page together wouldn't take long, and could include the info on how to disable the message.

**Comment by aaronsteers** [comment]
I think the fastest and clearest way, without creating tech debt is to create `meltano.com/get-help` (perhaps as a redirect to something like `meltano.com/get-help`. Providing a generic link seems less needing of a way to turn it off - versus a direct Slack CTA which is clearly the wrong CTA if folks are already on Slack or don't (for whatever reason) want to join.


![Image](https://user-images.githubusercontent.com/18150651/187547199-9b88c9c5-3aa2-403a-97cf-fd253a94527d.png)

**Comment by tayloramurphy** [comment]
@aaronsteers we have the melta.no URL which we can use for this. Let's put that in there and we can work on the redirect in parallel. Whoever picks this up can ping me in the PR and I'll make the redirect happen.

**Comment by aaronsteers** [comment]
@tayloramurphy - Can you confirm the above updated spec and updated issue description?

<img width="714" alt="image" src="https://user-images.githubusercontent.com/18150651/187789562-3d85cb66-a7af-41b9-99be-4e17130d4e71.png">

**Comment by aaronsteers** [comment]
This might be harder than we expect, since recent observations like

- https://github.com/meltano/meltano/issues/6684

**Comment by tayloramurphy** [comment]
@aaronsteers that spec looks good to me 👍

**Comment by tayloramurphy** [comment]
I've just updated the melta.no forwarding to go to https://docs.meltano.com/guide/troubleshooting

**Comment by edgarrmondragon** [comment]
@aaronsteers am I right to conclude that https://github.com/meltano/meltano/issues/6684 (and better exception handling, really) is blocking this?

**Comment by aaronsteers** [comment]
@edgarrmondragon - I do think that's probably a fair assertion. My understanding is that we can try to add this messaging in piecemeal, per error message - but that the better and long-term solve is to have a refactored "raise error to end user" code path.

Is that how you would think of this too?

**Comment by edgarrmondragon** [comment]
@aaronsteers yeah, that's something we need anyway
